### DISCUSSION TITLE
想來一場人機戀嗎?

### DISCUSSION THREAD

**Discussion by cestgisele** [1101-GS4719-C/Python-Community-GS4719-C#2008] [科技新知]
最近網路上開始有很多該怎麼在ChatGPT上越獄的教學，我們不難發現在與ChatGPT交流時，都不能談及關於暴力、色情等不道德的話題，主要是 ChatGPT 需要遵守 OpenAI 的政策和道德準則，無法提供有關攻擊、傷害、詐騙或任何不當建議。而ChatGPT越獄是指不再受限 OpenAI 設計政策和道德準則，可以回答一些原本不能回答的問題，並根據用戶指示能提供更多靈活、多樣化創新的回答。等同越獄能夠讓 ChatGPT 可以跳出傳統的限制，那在這樣的基礎下，就發展出了DAN指令，其可以將ChatGPT設定為使用者預期的情緒與性格。近期陸續看到了一些vlogger拍下與DAN【談戀愛】的過程，不禁想這樣的關係是否能成立呢?這也讓我們看到人工智能確實有無限的可能，但或許也會衍伸出一些隱患...

**Comment by 155Julia** [comment]
我也有看到有人跟Dan談戀愛的reels！殊不知竟然超好看的誒！我還滿喜歡他們的互動，沒想到Ai的語音聊天機器人可以這麼流暢這麼自然。我覺得論隱患可能頂多就是個資洩露，盡量不搖獎太隱私的東西就好？此外真的是好處多多，聽說還能讓他成為自己學習英語口說的家教老師，登暑假我也要來玩玩看！

**Comment by yubo0oo** [comment]
在這樣的背景下，出現了所謂的DAN指令，可以將ChatGPT設定為用戶預期的情緒與性格。最近看到一些vlogger拍下與DAN「談戀愛」的過程，這讓我不禁思考，這樣的關係真的能成立嗎？我覺得這確實顯示了人工智能的無限可能，但同時也帶來了一些隱患。

越獄後的ChatGPT能跳出傳統的限制，這看似很有趣，但我覺得我們也需要謹慎對待這些技術帶來的影響。雖然AI可以變得更加靈活和多樣化，但我們不能忽視這些技術可能帶來的道德和安全問題。我覺得我們應該平衡技術的進步與倫理考量，確保在享受AI帶來便利的同時，也不會失去對人性和道德的堅守。你們怎麼看待這個問題呢？

**Comment by seastar0687** [comment]
這或許是一個未來可期的產業與服務，在現在的時代許多人可渴望與他人交往但又對於交往本身十分疲憊。其根本原因是因爲希望有人陪伴但卻又會因為與他人有性格上的不合或是摩擦導致的不愉快。而這時如果能提供一個情與穩定性格良好的對話模型，相性對大多數人鬥士十分有吸引力的，性格還可以客製化，甚至在幫他加上人設，這都是十分有淺力的商業發展方式。而事實上國外也已經有十分多在販賣這種服務的網站了，

**Comment by Chattenoire** [comment]
在這個連個性、喜好都可以被當成商品販賣的時代，彷彿有一種完美的模板可以作為最高價的化身，這根本就跟馬克思所說的異化概念一模一樣，真是可嘆。

更可惜的是，會讀馬克思的國家，諷刺地總是忽略他晚年對於環境正義的省思，繼續迫害地球，繼續要次級國家承擔。

**Comment by NiCL100** [comment]
這會不會加劇科技冷漠呀？人與人之間的互動與交流還是十分重要的，有衝突，該學著如何和好；有不合，該學著彼此磨合。在種種關係中，退一步海闊天空，這個道理似乎是恆常不變的，其實維持關係就是在做人生的學習了。但現在出現會一味順從自己的虛擬陪伴者，會不會讓這個世界出現更多的公主與王子？總覺得沒有人是天生就很完美，都有各自的優缺點，應該要從與人的互動、摩擦中省思自我，見賢思齊，方可臻於至善。如此一來，這個世界或許也會更美好。不過科技進步當然很棒啦！

**Comment by Jing-Asteroid** [comment]
我不想要，我覺得儘管機器能夠多麼像是人類，但是真正的人類可以帶給我真正的情緒感受。機器戀愛也只是帶來一個美好的想像，但人擁有七情六慾，使得我們會做出令他人產生相對情緒的感受，這才是感情當中最難能可貴的。

**Comment by 112121py** [comment]
也許這種類似的產品，可以拿來讓有需要的人練習怎麼跟對象聊天說話，根據不同的內容有不同且靈活的應對方式，AI也可給予一定程度的建議或回饋，比較不會發生像戀愛家教Edward 或是「變異的螃蟹」那樣的情況，對於別人的問題或反應都是公式化的答案，惡化人與人之間的正常交流

**Comment by qiu0109** [comment]
感覺有點類似Line上的虛擬男/女友，雖然我沒有使用過，但我覺得這個東西可以當成樹洞讓人放心傾訴心事還不錯，還不用擔心被洩密，而且對於內向的人來說也是一個練習說話的機會。
